docs/components/concepts/clusters.md

```diff
@@ -6,72 +6,83 @@ description: "Learn more about the clusters available in your Camunda 8 plan."
+## Cluster size
 
-:::tip
+The cluster size defines the cluster performance and capacity.
 
-To prevent auto-pause, you can:
+Choose the cluster size that best meets your cluster environment requirements. See [sizing your environment](/components/best-practices/architecture/sizing-your-environment.md#sizing-your-runtime-environment).
 
-- Tag the cluster as `test`, `stage`, or `prod` instead of `dev`.
-- [Upgrade your Free Trial plan](https://camunda.com/pricing/) to a Starter, Professional, or Enterprise plan.
+- You can choose from four cluster sizes: 1x, 2x, 3x, 4x.
+- Each increase in size boosts cluster performance and adds capacity. Larger cluster sizes allow you to serve more workload.
+- Increased usage such as higher throughput or longer data retention requires a larger cluster size.
+- Each size increase uses one of your available cluster reservations.
+
+:::note
+
+Contact your Customer Success Manager to:
+
+- Increase the cluster size beyond the maximum 4x size. This requires custom sizing and pricing.
+- Increase the cluster size of an existing cluster.
 
 :::
 
-## Development clusters
+## Free Trial clusters
 
-Development clusters, available in the Starter and Enterprise plans, are recommended for development, testing, proof of concepts, and demos.
+Free Trial clusters have the same functionality as a production cluster, but are of a Basic type and 1x size, and only available during your trial period. You cannot convert a Free Trial cluster to a different kind of cluster.
 
-The way this type of cluster works varies depending on if you are using it in the Starter or the Enterprise plan.
+Once you sign up for a Free Trial, you are able to create one production cluster for the duration of your trial.
 
-### Development clusters in the Enterprise Plan
+When your Free Trial plan expires, you are automatically transferred to the Free Plan. This plan allows you to model BPMN and DMN collaboratively, but does not support execution of your models. Any cluster created during your trial is deleted, and you cannot create new clusters.
 
-Enterprise Plan users can purchase development clusters as part of their Enterprise subscription agreement. Deployment and execution of models (process instances, decision instances, and task users) are included at no extra cost for this type of cluster. Additionally, this type of cluster in the Enterprise plan follows the [standard data retention policy](/components/concepts/data-retention.md) and does not auto-pause when not in use.
+### Auto-pause
 
-Please [contact us](https://camunda.com/contact/) if you are an existing customer and would like to purchase a development cluster.
+Free Trial `dev` (or untagged) clusters are automatically paused eight hours after a cluster is created or resumed from a paused state. Auto-pause occurs regardless of cluster usage.
```

Comment:
@felix-mueller, just wondering: In the context of advanced offer there is no option for customers anymore to purchase 'dev' clusters, or? (The smallest  cluster is the base 1 hardware package. Does the auto-pause functionality then still make sense here?)